packages/profile-sync-controller/src/controllers/user-storage/encryption/cache.ts

```diff
@@ -28,13 +28,38 @@ const getPasswordCache = (hashedPassword: string) => {
  * @param salt - provide salt to receive cached key
  * @returns cached key
  */
+
+const passwordCacheMemo: Map<string, Map<string, Uint8Array>> = new Map();
+
+/**
+ * Returns a given memoized/cached derived key from a hashed password and salt
+ *
+ * @param hashedPassword - hashed password for cache lookup
+ * @returns cached key
+ */
+function getPasswordCacheMemo(hashedPassword: string): Map<string, Uint8Array> {
```

Comment:
Might need some help understanding the `getPasswordCacheMemo`. Since we are already caching by salt, this is salt+password hash?

Reply:
removed since its unecessary.